src/services/pool/mod.rs

```diff
@@ -578,7 +578,8 @@ impl PoolService {
         };
 
         if path.as_path().exists() {
-            return Err(PoolError::NotCreated("Already created".to_string()));
+            return Err(PoolError::AlreadyExists(
```

Comment:
At this point I am not sure from the error message what path the library is seeing verses what path I am passing in the config.

It would help to have the path that the rust library is seeing in the error message